### ISSUE TITLE
[Suggestion] StackableBeds and StackableBundles

### ISSUE THREAD

**Issue by God-ly** [gnembon/carpet-extra#192]
Make them stack up to 16 (empty bundles only).

**Comment by DragonEggBedrockBreaking** [comment]
This doesn't really fit in with the mod imo

**Comment by 1schlick** [comment]
Correct me if I'm wrong, but wouldn't this require players to have the mod as well?

**Comment by altrisi** [comment]
Yes in order to stack them (in case they stack in the inventory), not in order to have them stacked (just like the current stackableShulkerBoxes from main Carpet).

**Comment by God-ly** [comment]
> This doesn't really fit in with the mod imo

if not, then how does stackable shulker boxes fit?

**Comment by DragonEggBedrockBreaking** [comment]
> if not, then how does stackable shulker boxes fit?

Stackable empty shulkerboxes fits in because shulkerboxes are used a lot in technical play. Stackable beds isn't very vanilla-like, and stackable bundles is pointless, because technical minecrafters usually quickly go through the so called "early game", and bundles are basically pretty much useless.
